2696. Minimum String Length After Removing Substrings/solution.py

```diff
@@ -0,0 +1,50 @@
+class Solution:
+    # Given a string of uppercase English letters, repeatedly remove
+    # any instances of "AB" or "CD" from the string until no more instances
+    # exist. Return the length of the resulting string.
+    def minLength(self, s: str) -> int:
+
```

Comment:
I don't think you're supposed to use whitespace like this. At least not for at the beginning of functions, loops, and ifs!

Reply:
I looked up the Python style guide around blank lines and you are right. https://peps.python.org/pep-0008/#blank-lines